src/sample/stuDash.fxml

```diff
@@ -258,9 +258,19 @@
                         <Font size="18.0" />
                      </font>
                   </JFXButton>
+                  <Text fx:id="Messages" fill="#f70000" layoutX="273.0" layoutY="63.0" strokeType="OUTSIDE" strokeWidth="0.0" textAlignment="CENTER" wrappingWidth="384.0">
+                     <font>
+                        <Font name="System Bold" size="18.0" />
+                     </font>
+                  </Text>
+                  <JFXButton fx:id="contactButton" layoutX="690.0" layoutY="448.0" onAction="#CallUs" prefHeight="40.0" prefWidth="170.0" style="-fx-background-color: #EB7E31; -fx-cursor: hand;" text="Contact Us" textFill="WHITE" AnchorPane.leftAnchor="690.0" AnchorPane.topAnchor="448.0">
+                     <font>
+                        <Font size="18.0" />
+                     </font>
+                  </JFXButton>
```

Comment:
Put a pic about this

Reply:
Actually @YouQuan2007 , you can try to pass the name variable from studash.java to contactus.java. If the name variable is empty then cannot switch scene from contact us to courseregcontroller. There is no need to create another fxml (CallUs.fxml)